5주차/20922/20922_java_raccoon.java

```diff
@@ -0,0 +1,49 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayDeque;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Deque;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class Main {
+
+    static BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+
+    public static String input() {
+        try {
+            return reader.readLine().trim();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static void main(String[] args) {
+        String[] temp = input().split(" ");
+        int n = Integer.parseInt(temp[0]);
+        int k = Integer.parseInt(temp[1]);
+        int[] numbers = Arrays.stream(input().split(" "))
+                .mapToInt(Integer::parseInt)
+                .toArray();
+        int answer = 0;
+        Deque<Integer> deque = new ArrayDeque<>();
+        int[] counting = new int[100001];
+
+        for (int number : numbers) {
+            deque.offer(number);
+            counting[number] += 1;
```

Comment:
주관적인 의견이지만 이쪽이 더 의미가 명확하게 느껴져서 읽기 편한 것 같아요 ㅎㅎ

```suggestion
            counting[number]++;
```